app/src/main/kotlin/com/hippo/ehviewer/ui/screen/FavoritesScreen.kt

```diff
@@ -97,6 +98,7 @@ fun FavouritesScreen(navigator: DestinationsNavigator) = composing(navigator) {
     // Immutables
     val localFavName = stringResource(R.string.local_favorites)
     val cloudFavName = stringResource(R.string.cloud_favorites)
+    val animateItems by Settings.animateItems.collectAsState()
```

Comment:
Use a `@Composable` `Modifier` extension function like `Modifier.animateItemPlacementFix` to avoid collecting it here and there

Reply:
It will be collected multiple times on the same screen if doing so.